src/uct/tcp/tcp_ep.c

```diff
@@ -679,10 +684,9 @@ ucs_status_t uct_tcp_ep_connect_to_ep(uct_ep_h tl_ep,
         return uct_tcp_ep_connect(ep);
     }
 
-    if (ep->conn_state != UCT_TCP_EP_CONN_STATE_CONNECTED) {
+    if (ep->conn_state != UCT_TCP_EP_CONN_STATE_CONNECTED)  {
```

Comment:
extra space added
do we really need this 'if'? what would be the state if not CONNECTED?
why not always set UCT_TCP_EP_CONN_STATE_ACCEPTING?

Reply:
how can it be CONNECTED, if line 640 already checks it?

Reply:
good catch, removed